Infrastructure-Scripts/Get-AzStorageAccountConnectionString.ps1

```diff
@@ -0,0 +1,80 @@
+<#
+    .SYNOPSIS
+    Return either the primary or secondary connection string for a Storage Account and write to a Azure Pipelines variable.
+
+    .DESCRIPTION
+    Return either the primary or secondary connection string for a Storage Account and write to a Azure Pipelines variable.
+
+    .PARAMETER ResourceGroup
+    The name of the Resource Group that contains the Storage Account.
+
+    .PARAMETER StorageAccount
+    The name of the Storage Account.
+
+    .PARAMETER UseSecondaryKey
+    Boolean Switch to return the secondary connection string.
+
+    .PARAMETER OutputVariable
+    The name of the variable to be used by Azure Pipelines.
+
+    .EXAMPLE
+    .\Get-StorageAccountConnectionString.ps1 -ResourceGroup rgname -StorageAccount saname
+
+    Returns the Primary Storage Account connection string.
+
+    .EXAMPLE
+    .\Get-StorageAccountConnectionString.ps1 -ResourceGroup rgname -StorageAccount saname -UseSecondaryKey
+
+    Returns the Secondary Storage Account connection string.
+
+    .EXAMPLE
+    .\Get-StorageAccountConnectionString.ps1 -ResourceGroup rgname -StorageAccount saname -OutputVariable "CustomOutputVariable"
+
+    Returns the Primary Storage Account connection string and specifies a custom output variable to be used in VSTS.
+#>
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory = $true)]
```

Comment:
$true as a Constant would be PascalCase based on the guidelines, do we want $true and $false to be exempted?

Reply:
I like this issue discussion regarding constants and built-in variables: https://github.com/PoshCode/PowerShellPracticeAndStyle/issues/36. I've used tab completion to test the built-in casing that Microsoft maintains: i.e. there's a definite mixture:

$PSVersionTable
$true
$false
$PSHOME

I've updated the README naming convention table to exempt built-in constants / variables such as $true and $false.